### PR TITLE
Fix remote package links

### LINKED ISSUE #240: Broken link for Darwin installation
Following link is broken on this page (https://podman.io/getting-started/installation) is broken.

Remote Client
Latest remote client for MacOS (https://storage.googleapis.com/libpod-master-releases/podman-remote-latest-master-darwin-amd64.zip)

it would be great if you can fix the link and provide some detailed steps to setup podman on OSX. Thank you!

### PR DESCRIPTION
**Pull request by ashley-cui**
Fix windows link and move homebrew to correct section

Closes https://github.com/containers/podman.io/issues/240 & https://github.com/containers/podman.io/issues/246

Signed-off-by: Ashley Cui <acui@redhat.com>

### CODE REVIEW

**Comment by ashley-cui** [comment]
@rhatdan @TomSweeneyRedHat PTAL

Removed direct link to Mac release asset (better to install using Homebrew)

File: getting-started/installation.md [hunk 1]
```diff
@@ -12,11 +12,14 @@ title: Podman Installation
 Podman is a tool for running Linux containers.  You can do this from a MacOS
 desktop as long as you have access to a linux box either running inside of a
 VM on the host, or available via the network.  You need to install the remote
-client and then setup ssh connection information in the podman-remote.conf
-file.
+client and then setup ssh connection information.
```
Comment (TomSweeneyRedHat) [inline getting-started/installation.md:15]: any pointers on how to set this ssh connection up?

**Comment by TomSweeneyRedHat** [comment]
@ashley-cui if you could add a little more verbiage on how to set up the ssh connection, this will be GTG.  Thanks!

File: getting-started/installation.md (hunk 1 above)
Comment (ashley-cui) [reply getting-started/installation.md:15]: I'm gonna be honest, I have no idea and need to ask. Will find out and update

**Comment by rhatdan** [comment]
This is evolving and needs to be really updated once v2.0 is fully released.

**Comment by rhatdan** [comment]
Let's do the updates in a separate PR and merge this one.

**Pull request merged**